app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/service.tsx

```diff
@@ -72,6 +159,77 @@ export class Backend {
         }
         return response.model<Organization>();
       });
+
+  /** Returns a list of projects belonging to the current user, from the Cloud backend API. */
+  listProjects = (): Promise<Project[]> =>
+    this.get(LIST_PROJECTS_PATH)
+      .send()
+      .then(async (response) => {
+        if (response.status() === 401 || response.status() === 404) {
```

Comment:
consider either handling other statuses, or only accepting 200 OK

also consider extracting this check out to a function so it can be easily changed